src/CardsList.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext, useMemo, useEffect } from 'react';
+
+import { useFormContext } from 'react-hook-form';
+
+import Grid from '@mui/material/Grid';
+
+import Cards from './Cards';
+import aparts from './fixtures/data';
+import CurrencyData from './fixtures/CurrencyData';
+
+import Context from './Context';
+
+function CardsList() {
+  const { getValues } = useFormContext();
+
+  const ApartItem = getValues().currentApartment;
```

Comment:
Can be simplfied

const { 
  currentApartment: ApartItem,
  currentStreet: StreetName,
  currentRoomQuant: RoomItem
  ...
}  = getValues();

Reply:
Also, naming of the variables should be from lowerCase